src/main/java/Victor/Parser/Parser.java

```diff
@@ -0,0 +1,162 @@
+                    ui.displayBarrier();
+                    System.out.println("OK, I've marked this task as not done yet:");
+                    currentTask.unmarkAsDone();
+                    System.out.println(currentTask);
+                    ui.displayBarrier();
+                } catch (IndexOutOfBoundsException e) {
+                    ui.displayBarrier();
+                    System.out.println("Check how many items are in the list again. The number you gave is too high.");
+                    System.out.println("Can't unmark an item not in the list");
+                    System.out.println("The format to unmark a task is: unmark (task list number)");
+                    ui.displayBarrier();
+                } catch (NumberFormatException e) {
+                    ui.displayBarrier();
+                    System.out.println("Sorry, I'm only smart enough to find the task based on numbers.");
+                    System.out.println("Please give a number. If you refuse, too bad, this is all I can do.");
+                    ui.displayBarrier();
+                }
+            }
+            case "todo" -> {
+                try {
+                    Todo userTask = new Todo(inputList[1], false);
+                    currentTasks.addTask(userTask);
+                    ui.displayBarrier();
+                    System.out.println(userTask.toString());
+                    System.out.println("Now you have " + currentTasks.getSize() + " tasks in the list.");
+                    ui.displayBarrier();
+                } catch (IndexOutOfBoundsException e) {
+                    ui.displayBarrier();
+                    System.out.println("Sorry pal, but your description is empty.");
+                    System.out.println("Please redo the command and remember to add a description of the action");
+                    System.out.println("The format to schedule a todo task is: " + taskName.todo + " (Description)");
+                    ui.displayBarrier();
+                }
+            }
+            case "deadline" -> {
+                try {
+                    String[] differentParts = inputList[1].split("/", 2);
+                    String[] deadLine = differentParts[1].split(" ", 2);
+                    LocalDate deadDate = LocalDate.parse(deadLine[1]);
+                    Deadline userTask = new Deadline(differentParts[0], false, deadLine[1]);
+                    currentTasks.addTask(userTask);
+                    ui.displayBarrier();
+                    System.out.println(userTask.toString());
+                    System.out.println("Now you have " + currentTasks.getSize() + " tasks in the list.");
+                    ui.displayBarrier();
+                } catch (IndexOutOfBoundsException e) {
+                    ui.displayBarrier();
+                    System.out.println("Oh, you forgot to indicate when is the deadline or maybe you forgot the description.");
```

Comment:
Maybe consider using line breaks for longer lines?

Reply:
Fixed. Thanks